packages/web-components/src/components/rux-status/rux-status.scss

```diff
@@ -13,4 +13,10 @@
 }
 .status-wrapper {
     width: var(--spacing-3);
+    font-size: var(
```

Comment:
this is causing a pretty significant regression in the monitoring icon and monitoring progress icons. the status symbol is pushed much farther down. easiest fix might just be to override this in those components, ie 
```
.rux-advanced-status__status rux-status::part(status) { 
  line-height: 0; 
}
```

Reply:
Styles added.